milli/src/update/index_documents/transform.rs

```diff
@@ -293,6 +318,64 @@ impl<'a, 'i> Transform<'a, 'i> {
         Ok(documents_count)
     }
 
+    /// The counter part of `read_documents` that removes documents that may have been inserted into the transform previously.
+    pub fn remove_documents<FA>(
```

Comment:
Ah, maybe my comment was not super clear :thinking: 

The function does mark the element as « to delete » we need to, so once we enter `output_from_sorter` function, the document will be deleted from the database.

So, in the end, it can delete documents stored in the transform or from the db if it was never encountered before.

Reply:
Could you improve the comment details, please? By writing what you just wrote above for example.

Reply:
I added some details on how you can call it + why it is doing